module/VuFind/tests/unit-tests/src/VuFindTest/ILS/Driver/VirtuaTest.php

```diff
@@ -45,9 +45,10 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org Main Page
  */
-class VirtuaTest extends \VuFindTest\Unit\ILSDriverTestCase
+class VirtuaTest extends \PHPUnit\Framework\TestCase
```

Comment:
If you stop extending ILSDriverTestCase, it's probably worth introducing a copy of [testMissingConfiguration](https://github.com/vufind-org/vufind/blob/5638ccc11964657d969c63d1a54eb219c1f190a8/module/VuFind/src/VuFindTest/Unit/ILSDriverTestCase.php#L55) so we don't reduce our coverage.

Reply:
(I realize you did this because I suggested getting rid of the setup method -- and I still think that's a good idea -- but we might as well keep that test in place).

Reply:
Added a test for this one also